canonical-web-and-design/ubuntu.com
# Remove /kubernetes/partners page

## Done

- Remove /kubernetes/partners page - updated the title of the doc to 'OFFLINE'
- Redirect to /kubernetes
- Removed from the enterprise megamenu and update the copy doc
- Removed from the navigation (footer and secondary)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/kubernetes/partners
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that you are redirected to /kubernetes, that it isn't in the seconard nav or footer
- Open the enterprise mega menu, see that it isn't there (look for 'Rancher')
